flink-dist/src/main/resources/META-INF/NOTICE

```diff
@@ -20,6 +20,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.lz4:lz4-java:1.8.0
 - org.objenesis:objenesis:2.1
 - org.xerial.snappy:snappy-java:1.1.10.4
+- tools.profiler:async-profiler:2.9
```

Comment:
This change should be included in the 1st commit.

Reply:
Squashed with the first commit.